prac_06/used_cars.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical - Client code to use the Car class.
+Note that the import has a folder (module) in it.
+This is why we name our folders with no spaces or capitals, as valid module names.
+"""
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class."""
+    my_car = Car(180)
```

Comment:
not sure how your line will reference the car "name", will need to add a string literal, eg .. (Car, 180) for each object